app/views/campaigns/programs.html.haml

```diff
@@ -1,3 +1,5 @@
+- content_for :before_title, "#{@campaign.title} Programs — "
```

Comment:
To match the behavior of the header, which changes messages depending on whether the default is to call them 'Programs' or 'Courses', you can instead use the same message string from the h3 header: `t("#{Features.default_course_string_prefix}.campaign_courses", title: @campaign.title)`